docs/application/dotnet/samples/ui/overview.md

```diff
@@ -351,7 +355,7 @@ for TD:
 			<p><img alt="" height="225" src="media/tv3uicontrolsample.png" width="400" /></p>
 			</td>
 			<td>
-			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/TV/UIControlSample" target="_blank"><strong>(T) UIControl Sample (NUI)</strong></a></p>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/TV/NUI/UIControlSample" target="_blank"><strong>(T) UIControl Sample (NUI)</strong></a></p>
```

Comment:
Can we have a simpler image?

Four images put together, I am unable to get what's in the images.